hazelcast-spring/src/main/resources/hazelcast-spring-3.8.xsd

```diff
@@ -222,7 +222,7 @@
                                             Number of items in the ringbuffer. If no time-to-live-seconds is set, the size will
                                             always
                                             be equal to capacity after the head completed the first loop around the ring. This is
-                                            because no items are getting retired.
+                                            because no items are getting retired. The default value is 10000.
                                         </xs:documentation>
                                     </xs:annotation>
                                 </xs:attribute>
```

Comment:
I would. Can you suggest something, I am not sure if I can write something meaningful. Something like : 
> Custom classes implementing com.hazelcast.nio.serialization.DataSerializableFactory to be registered. These can be used to speed up serialization/deserialization of objects.